editor/js/Sidebar.Material.js

```diff
@@ -642,7 +642,7 @@ Sidebar.Material = function ( editor ) {
 			'emissive': materialEmissiveRow,
 			'specular': materialSpecularRow,
 			'shininess': materialShininessRow,
-			'program': materialProgramRow,
+			'vertexShader': materialProgramRow,
```

Comment:
Also, `.program` is defined for any material, not just `ShaderMaterial`. I guess that effect was also unintended as impossible to work correctly, not saying it wouldn't be great if we could plug custom shader code into any material one day :).